jaggery-modules/rxt/module/scripts/asset/asset.js

```diff
@@ -1667,6 +1711,49 @@ var asset = {};
         return asset;
     };
 
+    /***
+     * This method is used get the Current Lifecycle State Duration Meta Data.
+     * @param asset
+     * @returns {{}} json object that contains required metadata
+     */
+    AssetManager.prototype.getLCStateDurationDetails = function (asset) {
+        var details = {};
```

Comment:
Shall we refactor the method as follows:

var details = {} ;
details.lifecycleStateDuration = "time not available";

if ( !asset.id ) {
 log.error('Some error message here');
  return details;
}

if( !asset.lifecycle ) {
   log.error('Asset does not have a lifecycle ! ');
  return details ;
}

try {

} catch (e) {

}

return details;